src/main/java/fi/okm/jod/yksilo/config/SessionConfig.java

```diff
@@ -60,22 +54,7 @@ public RedisSerializer<Object> springSessionDefaultRedisSerializer() {
   }
 
   @Bean
-  public SessionRegistry sessionRegistry(RedisIndexedSessionRepository sessionRepository) {
-    return new SpringSessionBackedSessionRegistry<>(sessionRepository);
-  }
-
-  @Bean
-  @Order(2)
-  public SecurityFilterChain sessionFilterChain(HttpSecurity http, SessionRegistry sessionRegistry)
-      throws Exception {
-    return http.sessionManagement(
-            sessionManagement ->
-                sessionManagement.maximumSessions(1).sessionRegistry(sessionRegistry))
-        .build();
-  }
-
-  @Bean
-  @ConditionalOnBean(value = {AwsCredentialsProvider.class, Region.class})
+  @Profile("cloud")
```

Comment:
Should we try not to use a cloud profile in beans?

Reply:
We should try to avoid Profile, but [ConditionalOnBean](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/condition/ConditionalOnBean.html
) has the following note, so perhaps we should find another mechanism:
> The condition can only match the bean definitions that have been processed by the application context so far and, as such, it is strongly recommended to use this condition on auto-configuration classes only. If a candidate bean may be created by another auto-configuration, make sure that the one using this condition runs after.